packages/core/admin/admin/src/content-manager/history/pages/History.tsx

```diff
@@ -111,19 +109,18 @@ const HistoryPage = () => {
    * - slug for single types
    * - slug _and_ documentId for collection types
    */
-  if (!slug || (!documentId && collectionType === COLLECTION_TYPES)) {
+  if (!slug || (collectionType === COLLECTION_TYPES && !documentId)) {
     return <Navigate to="/content-manager" />;
   }
 
   // TODO: real error state when designs are ready
-  if (versionsResponse.isError || !versionsResponse.data || !layout || !schema || !permissions) {
-    return null;
+  if (versionsResponse.isError || !versionsResponse.data || !layout || !schema) {
+    return <Page.Error />;
```

Comment:
Would this lovely API have an empty state variant by any chance? (for when there's no selected version)

Reply:
It doesn't but you could make one called `NoData` the name just stretches further beyond content, really nice idea ⭐

Reply:
Sounds good, I can go ahead and set that up here unless either of you prefer it be done in a separate PR

Reply:
No no, do it in the context of here, we can critique it better here 👍🏼